### DISCUSSION TITLE
How to change the password?

### DISCUSSION THREAD

**Discussion by szkrstms** [josephdadams/TallyArbiter#644] [General]
I stopped the TallyArbiter from running. I modified the .local/share/TallyArbiter/config.json file.
 ... "users": [
  {
   "username": "admin",
   "password": "szkrstms",
   "roles": "admin".
  }, ...
I saved the changes and started TallyArbiter. However, the password was not encrypted, it kept the original text, so it won't let me in. What is the solution to encrypt the password(create a hash) that I can paste into the config file?

**Comment by Jocke4f** [comment]
The only way to set a specific password in TA 3.0.7 is when the user is added via the UI on Settings -> Users

Manually pasting an encrypted password into the config.json file defies the purpose of having encrypted passwords.

NB: If no config.json file is found is it recreated at TA startup with the two default users.

**Reply by szkrstms** [reply]
Tnx!
